ui/src/app/pages/signed-in/component.ts

```diff
@@ -84,21 +74,12 @@ export class SignedInComponent implements OnInit, OnDestroy, AfterViewInit {
   ngOnInit(): void {
     this.serverConfigService.getConfig().subscribe((config) => {
       this.profileLoadingSub = this.profileStorageService.profile$.subscribe((profile) => {
-        this.hasDataAccess = hasRegisteredAccess(profile.dataAccessLevel);
-        if (this.hasDataAccess) {
+        this.profile = profile as unknown as FetchProfile;
```

Comment:
What does this multiple-`as`-in-a-row do?

Reply:
This is converting from `generated`/angular, and `generated/fetch`. (Its a painful workaround to get around the typing.

Reply:
yuck.

Reply:
Yeah, I kinda hate it. Actually, let me see if the compiler is unhappy with removing it.

Reply:
Yeah, compiler complains without it.